examples/user_guide/07-Tabular_Datasets.ipynb

```diff
@@ -487,7 +487,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Data is often grouped in various ways, and the Dataset interface provides various means to easily compare between groups and apply statistical aggregates. We'll start by generating some synthetic data with two groups along the x-axis and 4 groups along the y axis."
+    "Data is often grouped in various ways, and the Dataset interface provides various means to easily compare between groups and apply statistical aggregates. We'll start by generating some synthetic data with two groups along the x-axis and 4 groups along the y-axis."
```

Comment:
Actually your comment had me intrigued: thanks for the link!
Re code changes, they were a better set of tick marks, a larger marker size,
and in one instance, better alignment (one of my pet peeves, I want to
see my indices at a glance, e.g.,
```
b = a[0][i-1] * a[1][i-1] +
      a[0][i   ] * a[1][i   ]
```